### PR TITLE
improvement: consume fiddle local code-gen apis

### PR DESCRIPTION
**Pull request by dsinghvi**

### CODE REVIEW

File: packages/cli/migrations/src/migrations/0.0.207/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { VersionMigrations } from "../../types/VersionMigrations";
+import UnionSinglePropertyKeyMigration from "./add-mode-to-draft-generators";
```
Comment (zachkirsch) [inline packages/cli/migrations/src/migrations/0.0.207/index.ts:2]: AddModeToDraftGeneratorsMigration

File: packages/cli/migrations/src/migrations/0.0.207/add-mode-to-draft-generators/migration.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    if (!YAML.isSeq(draftGenerators)) {
+        context.fail(`draft generators are not a list in ${filepath}`);
+        return;
+    }
+    draftGenerators.items.forEach((draftGenerator) => {
+        if (!YAML.isMap(draftGenerator)) {
+            context.fail(`draft generator is not an object in ${filepath}`);
+            return;
+        }
+        const name = draftGenerator.get("name");
```
Comment (zachkirsch) [inline packages/cli/migrations/src/migrations/0.0.207/add-mode-to-draft-generators/migration.ts:42]: def not worth changing, but for context I just remembered there's a boolean config somewhere that if it's true it'll return you the `Scalar` node with comments vs just the string

File: packages/cli/migrations/src/migrations/0.0.207/index.ts (hunk 1 above)
Comment (dsinghvi) [reply packages/cli/migrations/src/migrations/0.0.207/index.ts:2]: ah thanks

File: packages/cli/migrations/src/migrations/0.0.207/add-mode-to-draft-generators/migration.ts (hunk 2 above)
Comment (dsinghvi) [reply packages/cli/migrations/src/migrations/0.0.207/add-mode-to-draft-generators/migration.ts:42]: ah you're right

Comment (dsinghvi) [reply packages/cli/migrations/src/migrations/0.0.207/add-mode-to-draft-generators/migration.ts:42]: if i pass in true ill get the node as opposed to the `string`

**Pull request merged**